Library/Formula/oauth2_proxy.rb

```diff
@@ -0,0 +1,80 @@
+require "language/go"
+
+class Oauth2Proxy < Formula
+  desc "A reverse proxy that provides authentication with Google, Github or other provider"
```

Comment:
Done. (FWIW, there seemed to be a theme in the instructions about following precedent from other repositories, which is why I copied the github project 1-liner verbatim.)